arviz/plots/densityplot.py

```diff
@@ -73,10 +73,10 @@ def plot_density(
     hdi_prob : float
         Probability for the highest density interval. Should be in the interval (0, 1].
         Defaults to 0.94.
-    point_estimate : Optional[str]
+    point_estimate : str, optional
         Plot point estimate per variable. Values should be 'mean', 'median', 'mode' or None.
         Defaults to 'auto' i.e. it falls back to default set in ``rcParams``.
-    colors : Optional[Union[List[str],str]]
+    colors : str, optional
```

Comment:
`Union` indicates all the types inside its `[]` are valid. That is, `List[str]` and `str` are both valid. Therefore, the translation to person-friendly type (instead of type hints that target machines) is `str or list of str, optional` (the `optional` part is already right, I only include it for completeness)